libLanlGeoMag/Lgm_McIlwain_L.c

```diff
@@ -33,15 +33,15 @@
  *            \param[in]        u           Position (in GSM) to compute L-shell.
  *            \param[in]        Alpha       Pitch angle to compute L for. In degrees.
  *            \param[in]        Type        Flag to indicate which alogorithm to use (0=original McIlwain; else use Hilton's formula).
- *            \param[out]       I           The integral invariant, I that was computed along the way.
+ *            \param[out]       I0           The integral invariant, I that was computed along the way.
```

Comment:
Mostly you've been changing I to I1, and generally I recall I0 being used for a target I. Is this change consistent with any particular naming convention?

Reply:
There was none but there could be.  Since it's calculated here I_calc may be clearer... in other cases I0 was in use already and they could all be brought to some consistency.